PIPs/pip-39.md

```diff
@@ -3,7 +3,7 @@ pip: 39
 title: Batch Transfer
 description: Transfer Payload with Multiple Receivers
 author: Pactus Development Team <info@pactus.org>
-status: Draft
+status: Accepted
```

Comment:
PIP is not completed . Update the block number. We will discuss it in the next DEV meeting.

Reply:
Where put about block number of hard fork? I think after 260,000 block fork applied.